Weaver/WeaverTests/PCLRealmFreeAssemblyToProcess/PCLModuleLocator.cs

```diff
@@ -0,0 +1,27 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+namespace PCLRealmFreeAssemblyToProcess 
+{
+    /// <summary>
+    /// This class serves as nothing but a way to locate the assembly in the weaver test.
+    /// </summary>
+    public class PCLModuleLocator
```

Comment:
Because namespaces can be "used", essentially erasing any indication which class that is, unless you look at the usings. Calling it "RealmFreePCLModuleLocator" would still be clear.